plugins/module_utils/k8s/runner.py

```diff
@@ -57,6 +58,54 @@ def run_module(module) -> None:
         msg = "Failed to load resource definition: {0}".format(e)
         raise CoreException(msg) from e
 
+    select_all = module.params.get("select_all")
+    src = module.params.get("src")
+    resource_definition = module.params.get("resource_definition")
+    name = module.params.get("name")
+    state = module.params.get("state")
+
+    if select_all:
```

Comment:
I would really like to see @goneri's suggestion implemented here to pull this out into a separate function that will be easier to unit test.